qa/tasks/cephadm.py

```diff
@@ -11,6 +11,8 @@
 import uuid
 
 from io import BytesIO
+from typing import MutableMapping, Any
+import toml
```

Comment:
as discussed we also need to install `toml` on sepia. might want to get the PRs for teuthology targeting master and py2 merged first.

Reply:
https://github.com/ceph/teuthology/pull/1493

Reply:
is there a quick way to verify the code, before merging any PRs?

Reply:
thanks, we will need to backport this change to octopus once the mirror is up and running.

Reply:
> is there a quick way to verify the code, before merging any PRs?

you can run teuthology-suite with --teuthology-branch command to test it